react-rendering-lists/my-app/src/App.js

```diff
@@ -0,0 +1,17 @@
+import './App.css';
+import PokemonList from './PokemonList';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' },
+];
+
+function App() {
+  return <PokemonList />;
```

Comment:
better yet you can pass in the pokedex to the PokemonList component as prop and access it from the child